tests/test_setup_profile.py

```diff
@@ -55,9 +55,8 @@ def test_provided_keypair_copied_to_nanopub_dir(tmp_path: Path):
 
 
 def test_validate_orcid_id():
-    valid_ids = ['https://orcid.org/1234-5678-1234-5678', '']
-    for orcid_id in valid_ids:
-        validate_orcid_id(ctx=None, orcid_id=orcid_id)
+    valid_id = 'https://orcid.org/1234-5678-1234-5678'
+    assert validate_orcid_id(ctx=None, orcid_id=valid_id) == valid_id
     invalid_ids = ['https://orcid.org/abcd-efgh-abcd-efgh',
                    'https://orcid.org/1234-5678-1234-567',
                    'https://orcid.org/1234-5678-1234-56789',
```

Comment:
Very minor suggestion - maybe add an invalid ID that isn't a URI too (such as one of the form ```0000-0000-0000-0000```) because that is something people might conceivably enter.

Reply:
Good one, I've added it!